src/controllers/user.controller.js

```diff
@@ -0,0 +1,82 @@
+const userService = require('../services/user.service');
+
+const get = async (_req, res) => {
+  res.status(200).send(await userService.getAll());
+};
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  const newUser = await userService.createUser(name);
+
+  res.status(201).send(newUser);
+};
+
+const remove = async (req, res) => {
+  const { id } = req.params;
+  const user = await userService.getById(id);
+
+  if (!id) {
+    res.sendStatus(200);
+
+    return;
+  }
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  await userService.remove(+id);
+
+  res.sendStatus(204);
+};
+
+const update = async (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = await userService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.send(user);
```

Comment:
In the `update` function, if `name` is not a string, it should return a 400 status to indicate a bad request. Consider changing `res.send(user)` to `res.sendStatus(400)`.